addon/components/sl-radio-group.js

```diff
@@ -91,15 +88,15 @@ export default Ember.Component.extend( InputBased, TooltipEnabled, {
     /**
      * Initialize the group-wide options and setup child radio buttons
      *
```

Comment:
DocBlock should contain @throws {ember.assert} and contain a description of condition causing if it makes sense to do so